internal/host/validator.go

```diff
@@ -425,366 +435,363 @@ func (v *validator) areTangServersReachable(c *validationContext) ValidationStat
 	return boolValue(response.IsSuccess)
 }
 
-func (v *validator) diskEncryptionRequirementsSatisfied(c *validationContext) ValidationStatus {
+func (v *validator) diskEncryptionRequirementsSatisfied(c *validationContext) (ValidationStatus, string) {
```

Comment:
The main reason to do the change is not to perform checks twice, but we are doing double checks here.
Do you intend to refactor this function (and others), to eliminate double checks ?

Reply:
I have made changes that should address this.